contentcuration/contentcuration/static/js/exercise_creation/models.js

```diff
@@ -0,0 +1,71 @@
+var Backbone = require("backbone");
+var _ = require("underscore");
+
+var ExerciseModel = Backbone.Model.extend({
+	defaults: {
+		title: "Exercise"
+	},
+
+	url: function() {
+		return window.Urls["exercise-list"]() + this.get("id");
+	}
+});
+
+var ExerciseCollection = Backbone.Collection.extend({
+	model: ExerciseModel,
+
+	url: function() {
+		return window.Urls["exercise-list"]();
+	}
+});
+
+var AssessmentItemModel = Backbone.Model.extend({
+
+	urlRoot: function() {
+		return window.Urls["assessmentitem-list"]();
+	},
+
+	defaults: {
+		question: "Question Text",
+		answers: "[]"
+	},
+
+	initialize: function () {
+		if (typeof this.get("answers") !=="object") {
```

Comment:
If it is already a collection, yep.

On Thu, 10 Sep 2015 19:13 Michael Gallaspy <notifications@github.com> wrote:

> In contentcuration/contentcuration/static/js/exercise_creation/models.js
> <https://github.com/fle-internal/content-curation/pull/15#discussion_r39235871>
> :
>
> > +	}
> > +});
> > +
> > +var AssessmentItemModel = Backbone.Model.extend({
> > +
> > +	urlRoot: function() {
> > +		return window.Urls["assessmentitem-list"]();
> > +	},
> > +
> > +	defaults: {
> > +		question: "Question Text",
> > +		answers: "[]"
> > +	},
> > +
> > +	initialize: function () {
> > +		if (typeof this.get("answers") !=="object") {
>
> You're trying to determine if it should be deserialized?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fle-internal/content-curation/pull/15/files#r39235871>
> .
>